app/services/integrations/aggregator/taxes/credit_notes/create_service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Integrations
+  module Aggregator
+    module Taxes
+      module CreditNotes
+        class CreateService < Integrations::Aggregator::Taxes::Invoices::BaseService
```

Comment:
I thought we want to have CreditNotes service to be separate from Invoice service, and in this case shouldn't we have a separate Base service for CreditNotes? or if it will be reused, extract the base from invoices, and have `Integrations::Aggregator::Taxes::BaseService`?

Reply:
Well, the point was to have a dedicated service for credit notes so that we can centralise credit note logic and put it from the start in the right place. I wouldn't say it is problematic to inherit from invoices base service since single-responsibility principle is still satisfied because the main goal was just to have this dedicated service. However, if you want, we can also structure a bit different parent classes. Since everything is actually inherited from `Integrations::Aggregator::BaseService` maybe we can upgrade a bit the full stack of inheritance (i know few other inheritance related parts that can be updated), but since it could be bigger change, let's do it as the last step of this story but before the QA. How does it sound?